src/schema/schema_create.c

```diff
@@ -985,9 +1104,18 @@ __schema_create(WT_SESSION_IMPL *session, const char *uri, const char *config)
      * back it all out.
      */
     WT_RET(__wt_meta_track_on(session));
-    if (import)
+    if (import) {
         F_SET(session, WT_SESSION_IMPORT);
 
+        if (__wt_config_getones(session, config, "import.metadata_file", &cval) == 0 &&
+          cval.len != 0 && (cval.type == WT_CONFIG_ITEM_STRING || cval.type == WT_CONFIG_ITEM_ID)) {
+            WT_ERR(__wt_strndup(session, cval.str, cval.len, &export_file));
+            import_list.uri = uri;
+            WT_ERR(__get_uri_suffix(session, uri, &import_list.uri_suffix));
```

Comment:
Again we should be able to assume well-formed names.

Reply:
Done